packages/generator/src/mutation/templates/MutationAddWithSchema.js.template

```diff
@@ -37,10 +37,17 @@ export default mutationWithClientMutationId({
       <%_ } -%>
     } = args;
 
+    // Create new model
+    const <%= camelCaseName %> = await new <%= name %>({
+        <%_ for (field of schema.fields) { -%>
```

Comment:
These fields should also have a value, can be `'Example value'` for now and the user fulfils them later.